lib/easypost/address.rb

```diff
@@ -2,6 +2,10 @@ module EasyPost
   class Address < Resource
     attr_accessor :message # Backwards compatibility
 
+    def self.raise_verification_failure
+      raise Error.new("Unable to verify address.")
+    end
```

Comment:
Not really sure why this was a function to begin with. Can you just inline this in both places?